BUILDING.md

```diff
@@ -48,22 +48,23 @@ expects few pre requisites to be installed before hand.
     If any of the defaults needs to be changed, the parameter needs to be specified explicitly while running the script.
     
     - If you want to build just the documentation, without the library.
-        ```bash
-        ./build.sh -b false
-        ```
+      ```bash
+      ./build.sh -b false
+      ```
         
     - If you want to build just the library but aren't interested in building docs.
-        ```bash
-        ./build.sh -d false
-        ```
+      ```bash
+      ./build.sh -d false
+      ```
         
     - If you just want to build the library, and you have python interpreter installed at /usr/lib/python
-        ```bash
-        ./build.sh -d false -p /usr/lib/python
-        ```
+      ```bash
+      ./build.sh -d false -p /usr/lib/python
+      ./build.sh -d false -p python3.6
+      ```
         
     - And so on depending on your requirements.
 
-Once done,
+  Once done,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/manual/JanusGraph/janusgraph-python/pullRequest?prid=4792911)